Dockerfile.debian

```diff
@@ -0,0 +1,102 @@
+# syntax = docker/dockerfile:1.2
+FROM docker.io/library/golang:1.19-bullseye AS builder
+
+RUN apt update
+RUN apt install -y build-essential git bash ca-certificates libstdc++6
```

Comment:
Are you sure libstdc++6 is needed and why 6

Reply:
i copied the deps from alpine - the matching dependency in terms of most compatibililty for libstdc++ is libstdc++6 (glibc6). Not 100% if it is needed for runtime, i assumed it was there in the alpine one for a reason though. 

the other option in debian is libstdc++5 (glibc3.3)